modules/sync/syncs/httpsynchronization.cpp

```diff
@@ -97,24 +98,47 @@ void HttpSynchronization::start() {
     }
     _state = State::Syncing;
 
-    if (hasSyncFile()) {
+    if (checkSyncFile()) {
```

Comment:
Since the meaning of the return value is not very obvious (what does it mean that it is checked to be true?) it would be better to store the return value in a variable that has a descriptive name; for example
```cpp
bool isSynced = checkSyncFile();
if (isSynced) {
```

maybe better to name the function something else to make it obvious what the `true`/`false` returns are